src/cluster.c

```diff
@@ -3869,7 +3869,20 @@ void clusterLogCantFailover(int reason) {
         break;
     }
     lastlog_time = time(NULL);
-    serverLog(LL_WARNING,"Currently unable to failover: %s", msg);
+    serverLog(LL_NOTICE,"Currently unable to failover: %s", msg);
+    
+    static int last_vote;
+    static int last_quorum;
+    int cur_vote = server.cluster->failover_auth_count;
+    int cur_quorum = (server.cluster->size / 2) + 1;
+    /* Emits a log when an election is attemped but doesn't succeed.
+       But don't log if we have same vote and quorum as last time. */
+    if (reason == CLUSTER_CANT_FAILOVER_WAITING_VOTES) {
+        if (last_vote == cur_vote && last_quorum == cur_quorum) return;
```

Comment:
You can have those 2 conditions in a single if statement.

Reply:
We still need `if (reason == CLUSTER_CANT_FAILOVER_WAITING_VOTES)` for the ongoing part.